docker-compose.yml

```diff
@@ -104,7 +104,7 @@ services:
       - FLAGD_PORT
       - VALKEY_ADDR
       - OTEL_EXPORTER_OTLP_ENDPOINT
-      - OTEL_RESOURCE_ATTRIBUTES
+      - OTEL_RESOURCE_ATTRIBUTES=${OTEL_RESOURCE_ATTRIBUTES},service.instance.id=inst-12345 # workaround for https://github.com/open-telemetry/opentelemetry-demo/issues/2399
```

Comment:
@matt-hensley FYI here is a workaround waiting for a fix. ref:
* https://github.com/open-telemetry/opentelemetry-demo/issues/2399

Reply:
@cyrille-leclerc the fix was merged, would you be able to update this PR?

Reply:
Thanks, fixed.